packages/volto/src/components/manage/Edit/Edit.jsx

```diff
@@ -346,11 +346,18 @@ class Edit extends Component {
               <>
                 <Helmet
                   title={
-                    this.props?.schema?.title
+                    this.props?.content?.['@components']?.breadcrumbs?.items
```

Comment:
@Faakhir30 Why do you take the `title` from breadcrumbs? Isn't is available in `this.props.content`?

Reply:
My bad,  overlooked `title` field in `props.content`, fixed now